rep-2010.rst

```diff
@@ -220,7 +219,34 @@ And the definitions for ``example_interfaces/srv/AddTwoInts_Reponse`` is,
 Configuration
 -------------
 
-TODO: Describe what configuration options there are and how we plan to expose them (e.g. with ROS parameters and/or environment variables).
+Configuration of service introspection features will be done through ROS parameters.
+
+.. code-block:: yaml
```

Comment:
I think it would be better to just list out parameters and their descriptions in rst, and we can conclude the section with a concrete example in YAML.

Reply:
See [83f0841](https://github.com/jacobperron/rep/pull/2/commits/83f084112210ca4a6bb279f11ef93a5311b783db)